lib/Robo/Plugin/Commands/TestEnvironmentCommands.php

```diff
@@ -224,8 +224,9 @@ private function installUnixEnvironmentVariables(array $opts)
                 if (stristr($line, $optionKeyReplaced) !== false) {
                     $self->say('Removed: ' . $optionKeyReplaced);
                     return '';
+                } else {
```

Comment:
This else would actually be unnecessary, you can remove it for lower complexity numbers.